tensorflow/contrib/tensorrt/custom_plugin_examples/test/plugin_test.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Please add a bazel rule to this test. To make it simpler, I think moving the test to the custom_plugin_example dir is sufficient.